ebuild_generator/ebuild.py

```diff
@@ -260,6 +264,8 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
 
         # source configuration
         ret += "src_configure() {\n"
+        if self.name == 'opencv3':
+            ret += "    filter-flags \'-march=*\' \'-mtune=*\'\n"
```

Comment:
I don't think you need to escape the single quotes, since they're in a double quote string. Also, can you verify that this needs '='?